net module compile error

I just git clone the net module on centos 7, and run 'make rpm', but it's fail.
n file included from isolator/network_isolator.cpp:68:0:
isolator/network_isolator.hpp:89:26: error: 'ContainerLaunchInfo' is not a member of 'mesos::slave'
   process::Future<Option<mesos::slave::ContainerLaunchInfo>> prepare(
                          ^
isolator/network_isolator.hpp:89:26: error: 'ContainerLaunchInfo' is not a member of 'mesos::slave'
isolator/network_isolator.hpp:89:40: error: template argument 1 is invalid
   process::Future<Option<mesos::slave::ContainerLaunchInfo>> prepare(
                                        ^
isolator/network_isolator.hpp:89:59: error: template argument 1 is invalid
   process::Future<Option<mesos::slave::ContainerLaunchInfo>> prepare(
                                                           ^
isolator/network_isolator.hpp:91:13: error: 'ContainerConfig' in namespace 'mesos::slave' does not name a type
       const mesos::slave::ContainerConfig& containerConfig);
             ^
isolator/network_isolator.hpp:91:44: error: ISO C++ forbids declaration of 'containerConfig' with no type [-fpermissive]
       const mesos::slave::ContainerConfig& containerConfig);
                                            ^
isolator/network_isolator.hpp:149:34: error: 'ContainerLaunchInfo' is not a member of 'mesos::slave'
   virtual process::Future<Option<mesos::slave::ContainerLaunchInfo>> prepare(
                                  ^
isolator/network_isolator.hpp:149:34: error: 'ContainerLaunchInfo' is not a member of 'mesos::slave'
isolator/network_isolator.hpp:149:48: error: template argument 1 is invalid
   virtual process::Future<Option<mesos::slave::ContainerLaunchInfo>> prepare(

--- a/packages/Dockerfile
+++ b/packages/Dockerfile
@@ -67,7 +67,7 @@ RUN wget http://search.maven.org/remotecontent?filepath=com/google/protobuf/prot
 WORKDIR /root/rpmbuild/SOURCES/
 # Note: Changes to mesos source URL should also be reflected in the
 # mesos_github_tag var in packages/mesos.spec
-RUN wget https://github.com/apache/mesos/archive/0.26.0.tar.gz
+RUN wget https://github.com/apache/mesos/archive/0.27.0.tar.gz
 ADD ./sources/mesos/ /root/rpmbuild/SOURCES/
 
 # MESOS

--- a/packages/mesos.spec
+++ b/packages/mesos.spec
@@ -1,5 +1,5 @@
 Name:          mesos
-Version:       0.26.0
+Version:       0.27.0
 Release:       1.custom
 Summary:       Cluster manager for sharing distributed application frameworks
 License:       ASL 2.0
@@ -7,7 +7,7 @@ URL:           http://mesos.apache.org/
 
 ExclusiveArch: x86_64
 
-%define mesos_github_tag 0.26.0
+%define mesos_github_tag 0.27.0
 
 Source0:       %{mesos_github_tag}.tar.gz
 Source1:       %{name}
@@ -202,6 +202,10 @@ mkdir -p -m0755 %{buildroot}/%{_var}/lib/%{name}
 %doc LICENSE NOTICE
 %{_libdir}/libmesos*.so
 %{_libdir}/libfixed_resource_estimator-%{version}.so
+%{_libdir}/libload_qos_controller-0.27.0.so
+%{_libdir}/libload_qos_controller.so
+%{_libdir}/liblogrotate_container_logger-0.27.0.so
+%{_libdir}/liblogrotate_container_logger.so
 %{_bindir}/mesos*
 %{_sbindir}/mesos-*
 %{_datadir}/%{name}/
@@ -215,7 +219,7 @@ mkdir -p -m0755 %{buildroot}/%{_var}/lib/%{name}
 %{_sysconfdir}/%{name}-master/*
 %{_sysconfdir}/%{name}-slave
 %{_sysconfdir}/%{name}/*
-
+%{_includedir}/picojson.h
 
 ######################
 %files devel
@@ -265,6 +269,9 @@ exit 0
 
 
 %changelog
+* Tue Feb 16 2016 Dan Osborne <daniel.osborne@metaswitch.com> - 0.27.0-1.custom
+- Build mesos 0.27.0
+
 * Tue Dec 22 2015 Dan Osborne <daniel.osborne@metaswitch.com> - 0.26.0-1.custom
 - Build mesos 0.26.0
 

--- a/packages/net-modules.spec
+++ b/packages/net-modules.spec
@@ -1,5 +1,5 @@
 Name:          net-modules
-Version:       0.26
+Version:       0.2.0
 Release:       1.custom
 Summary:       Network isolation modules for Apache Mesos
 License:       ASL 2.0
@@ -53,6 +53,9 @@ install %{SOURCE2} %{buildroot}%{_sysconfdir}/mesos-slave/
 %{_sysconfdir}/mesos-slave/hooks
 
 %changelog
+* Tue Feb 16 2016 Dan Osborne <daniel.osborne@metaswitch.com> - 0.2.0-1.custom
+- Build netmoudules 0.2.0
+
 * Tue Dec 22 2015 Dan Osborne <daniel.osborne@metaswitch.com> - 0.26-1.custom
 - Build mesos 0.26.0
 

